cli/src/pcluster/resources/imagebuilder/parallelcluster_test.yaml

```diff
@@ -264,6 +264,19 @@ phases:
               done
               echo "SchedulerPlugin Group test passed"
 
+      - name: SchedulerPluginEventHandlerScript
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -vx
+              Script=/usr/local/sbin/invoke-scheduler-plugin-event-handler.sh
+              if [ -e ${Script} ]; then
```

Comment:
check on `-e` is to generic, because invoke-scheduler-plugin-event-handler.sh could have been a directory and the test would have passed. You can use `-x` so we check file exists and is executable.

Reply:
Change to use -x